components/SingleChat.js

```diff
@@ -6,6 +6,7 @@ import { db } from "../Firebase";
 
 import Layout from "../constants/Layout";
 import { fetchMessages, postMessage } from "../store";
+import { fetchEarlierMessages } from "../store/messages";
```

Comment:
Can you combine this import with the one on line 8?

Reply:
yeah good catch, I was debugging something and moved it in case it was the problem but I can combine it